.env.demo

```
@@ -128,158 +107,87 @@ PLATFORM_SEED=000000000000000000000000Steward1
-
-# Keycloak admin URL
 KEYCLOAK_ADMIN_URL=http://localhost:8080
-
-# Keycloak master realm name
 KEYCLOAK_MASTER_REALM=master
-
-# Keycloak management client ID
-KEYCLOAK_MANAGEMENT_CLIENT_ID=adminClient
-
-# Keycloak management client secret
+KEYCLOAK_MANAGEMENT_CLIENT_ID=credeblClient # Keycloak client created for platform management in keycloak.
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KEYCLOAK_MANAGEMENT_CLIENT_ID key should go before the KEYCLOAK_MASTER_REALM key

(UnorderedKey)